docs/modules/ROOT/pages/reference/extensions/vertx-http.adoc

```diff
@@ -55,3 +55,25 @@ You will also need to enable serialization for the exception classes that you in
 ----
 @RegisterForReflection(targets = { IllegalStateException.class, MyCustomException.class }, serialization = true)
 ----
+
+[id="extensions-vertx-http-additional-camel-quarkus-configuration"]
+== Additional Camel Quarkus configuration
+
+[id="extensions-vertx-http-configuration-allowjavaserializedobject-option-in-native-mode"]
+== allowJavaSerializedObject option in native mode
+
+When using the `allowJavaSerializedObject` option in native mode, the support of serialization might need to be enabled.
+Please, refer to the xref:user-guide/native-mode.adoc#serialization[native mode user guide] for more information.
+
+Some classes are registered https://github.com/apache/camel-quarkus/blob/main/extensions-core/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelSerializationProcessor.java[by default].
+In case another class is to be serialized, then explicit registration as below might be needed:
+[source,java]
+----
+@RegisterForReflection(targets = { MyCustomContent.class }, serialization = true)
```

Comment:
Do we need an example for this ?

Reply:
That's a good point. Maybe it's redundant with https://camel.apache.org/camel-quarkus/2.16.x/user-guide/native-mode.html#serialization ? So that we could remove line 68 -> 72 ?